src/System.Security.Permissions/src/System/Security/ISecurityEncodable.cs

```diff
@@ -6,5 +6,7 @@ namespace System.Security
 {
     public partial interface ISecurityEncodable
     {
+        void FromXml(System.Security.SecurityElement e);
```

Comment:
yes

---
In reply to: [76333927](https://github.com/dotnet/corefx/pull/11092#discussion_r76333927) [](ancestors = 76333927)